config/samples/nodefeaturediscoveries.nfd.k8s-sigs.io.yaml

```
@@ -0,0 +1,157 @@
+apiVersion: nfd.k8s-sigs.io/v1
+kind: NodeFeatureDiscovery
+metadata:
+  annotations:
+    helm.sdk.operatorframework.io/uninstall-wait: "true"
+  name: nodefeaturediscovery-instance
+  namespace: node-feature-discovery-operator
```

Hmm, I think here (for the operand) we'd like to use a different namespace(?)
```suggestion
  namespace: node-feature-discovery
```